js/main.js

```diff
@@ -94,13 +113,37 @@ const getRandomInt = (min, max) => {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 };
 
-const getDeclension = (number, titles) => {
-  const cases = [2, 0, 1, 1, 1, 2];
-  return titles[(number % 100 > 4 && number % 100 < 20) ? 2 : cases[(number % 10 < 5) ? number % 10 : 5]];
+// const getDeclension = (number, titles) => {
+//   const cases = [2, 0, 1, 1, 1, 2];
+//   return titles[(number % 100 > 4 && number % 100 < 20) ? 2 : cases[(number % 10 < 5) ? number % 10 : 5]];
+// };
+
+const getMainMapPinCoordinateX = () => {
+  return Number.parseInt(mapPinMain.style.left, 10) + MainPinSize.WIDTH / 2;
```

Comment:
`parseInt` доступен и без `Number`
```suggestion
  return parseInt(mapPinMain.style.left) + (MainPinSize.WIDTH / 2);
```
Хорошим тоном является группировать операнды в скобки есть имеется несколько операторов, иначе сложность понимания возрастает вместе с количеством операндов. Также это грозит не очевидными ошибками из-за приоритетности одних операций над другими